index.html

```diff
@@ -1,657 +1,1295 @@
+                                  <div id="container-feb5484fc2" class="cmp-container">
+                                    <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+                                      <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                        <div id="title-0a2b1292c9" class="cmp-title">
+                                          <h4 class="cmp-title__text">All Resources</h4>
+                                        </div>
+                                      </div>
+
+                                      <div class="text aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                        <div id="text-cd72b55a7f" class="cmp-text">
+                                          <p>Lorem ipsum dolor sit amet, consetetur sadip.</p>
+                                        </div>
+                                      </div>
+                                    </div>
+                                  </div>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
                         </div>
+                      </div>
                     </div>
+                  </div>
                 </div>
+              </div>
             </div>
-        </section>
-        <section class="newsletter-bar">
-            <div class="container">
-                <div class="title">
-                    <div data-cmp-data-layer="" id="" class="title-wrap">
-                        <h3>Join Thousands of Leaders Leaving a Lasting Impact</h3>
-                    </div>
+          </div>
+        </div>
+
+        <div class="container responsivegrid newsletter-bar aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="news-container" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 news-grid">
+              <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="title-822879a287" class="cmp-title">
+                  <h3 class="cmp-title__text">Join Thousands of Leaders Leaving a Lasting Impact</h3>
                 </div>
-                <div class="btn-wrap">
-                    <div class="button">
-                        <a href="" id="" class="btn-link" data-cmp-clickable="" data-cmp-data-layer="">
-                            <span class="btn-text">Subscribe to Our Newsletter</span>
-                        </a>
+              </div>
+
+              <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid button_block">
+                <div id="container-fcf0395128" class="cmp-container">
+                  <div  class="aem-Grid aem-Grid--12 aem-Grid--default--12 button-wrap">

```

Comment:
Either `button-wrap` will go on a new `div` below this line or along with `button_block`. This placement of this class is illegal.